src/styles/globals.scss

```diff
@@ -0,0 +1,36 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --white: #ffffff;
+  --black: #000000;
+}
+
+@media (max-width: 1080px) {
+  html {
+    font-size: 93.75%;
+  }
+}
+
+@media (max-width: 720px) {
+  html {
+    font-size: 87.5%;
+  }
+}
```

Comment:
Do we have to set different font sizes? I guess that we can stick with the browser font size, normally it is 16px, but it can vary. This way we can use `rem`. 
Normally in small devices, the default font size is smaller, so we don't have to worry about them.
You can define the font size on the body to be `0.875rem` (14px).